src/util/FactorUtil.js

```diff
@@ -202,17 +202,28 @@ function (Okta, TimeUtil) {
       return Okta.loc(propertiesString, 'login', [displayableTime.time]);
     };
 
-    if (policy.complexity) {
+    var setExcludeAttributes = function (policyComplexity) {
+      var excludeAttributes = policyComplexity.excludeAttributes;
```

Comment:
We're slightly changing the behavior here because this code assumes policy.complexity is set - is that always the case?

Reply:
That's a good point. I asked the backend dev and it might be that it is. But because it is a public API it is probably better to revert this

Reply:
yeah, all of this logic should be inside the ```if (policy.complexity)``` below, and maybe just have a helper method to find if an specific attribute is excluded...

Reply:
@mauriciocastillosilva-okta  Do my most recent changes make sense? I looked at it for awhile, but decided to leave setExcludeAttributes because 1. it was clear and 2. it would ignore any other attributes that may be in that array. Please let me know if you see some way for improvement.

Reply:
There is only one thing then: in this method we are adding an```excludeFirstName``` and ```excludeLastName``` to the original ```policy``` object, can we make sure then that does not affect anything else?
As a nit, in line 227 we have ```if (complexityValue <= 0) { // to skip 0 and -1``` which works with true/false, but it is a little odd...